xml/System.IO.Pipelines/PipeReader.xml

```diff
@@ -141,6 +146,9 @@ The <xref:System.IO.Pipelines.ReadResult.Buffer> previously returned from <xref:
 
 The examined data communicates to the pipeline when it should signal more data is available.
 
+The examined parameter should be greater than or equal to the examined position in the previous call to AdavanceTo. Otherwise <xref:System.InalidOperationException> will be thrown.
+The issue https://github.com/dotnet/runtime/issues/37176 discusses this.
```

Comment:
Again, I think we should remove the link to the issue.

Reply:
I think we can leave out the link.